app/views/smart_answers/shared/_previous_answers.html.erb

```diff
@@ -1,54 +1,62 @@
-        <%= render "govuk_publishing_components/components/heading", {
-          text: "Change your answers",
-          heading_level: 2,
-          font_size: "l",
-          margin_bottom: 6
-        } %>
-      <% else %>
-        <%= render "govuk_publishing_components/components/heading", {
-          text: "Your answers",
-          heading_level: 2,
-          font_size: "m",
-          margin_bottom: 4
-        } %>
-      <% end %>
+  <% if local_assigns[:hide_previous_answers] %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Change your answers",
+      heading_level: 2,
+      font_size: "l",
+      margin_bottom: 6
+    } %>
+  <% else %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Your answers",
+      heading_level: 2,
+      font_size: "m",
+      margin_bottom: 4
+    } %>
+  <% end %>
 
-      <p class="govuk-body">
-      <%= link_to "Start again",
-        restart_flow_path(@presenter),
-        :class => "govuk-link",
-        :data => {
-          module: "gem-track-click",
-          "track-action": "Start again",
-          "track-category": "StartAgain",
-          "track-label": @presenter.current_node.title
-        } %>
-      </p>
-    </caption>
+  <%= tag.p class: "govuk-body" do %>
+    <%= link_to "Start again",
+      restart_flow_path(@presenter),
+      class: "govuk-link",
+      data: {
+        module: "gem-track-click",
+        "track-action": "Start again",
+        "track-category": "StartAgain",
+        "track-label": @presenter.current_node.title
+      } %>
+  <% end %>
 
-    <% unless local_assigns[:hide_previous_answers] %>
-      <thead class="govuk-table__head">
-        <tr class="govuk-table__row">
-          <th scope="col" class="govuk-table__header">Question</th>
-          <th scope="col" class="govuk-table__header">Answer</th>
-          <th scope="col" class="govuk-table__header">Change answer</th>
-        </tr>
-      </thead>
-      <tbody class="govuk-table__body">
-        <% @presenter.collapsed_questions.each_with_index do |question, number_questions_so_far| %>
-          <%= render(
-              'smart_answers/shared/previous_answer',
-              question: question,
-              accepted_response: @presenter.accepted_responses[number_questions_so_far],
-              question_link: @presenter.change_collapsed_question_link(number_questions_so_far + 1, question),
-            )
-          %>
-        <% end %>
-      </tbody>
-    <% end %>
-  </table>
+  <% unless local_assigns[:hide_previous_answers] %>
+    <% items = @presenter.collapsed_questions.each_with_index.map { |question, number_questions_so_far|
+      accepted_response = @presenter.accepted_responses[number_questions_so_far]
+
+      if question.multiple_responses?
+        value = render "govuk_publishing_components/components/list", {
+          items: question.response_labels(accepted_response)
+        }
```

Comment:
Is the intended output a `<ul>`? I don't think this works because the accepted responses is always a string, not an array of responses. For questions where `multiple_responses? == true`, the output is string with the responses joined with a `,` e.g. `"response1, response2, response3"` not `["response1", "response2", "response3"]`.

Reply:
Yes, the intention is to render a list of items when we deal with multiple items. `response_labels` seems to be [always returning an array as it splits by `,`](https://github.com/alphagov/smart-answers/blob/main/app/presenters/checkbox_question_presenter.rb#L2-L10).

Reply:
🤦🏻 Sorry you're right!! I was getting confused!